zetaclient/logs/README.md

```diff
@@ -0,0 +1,21 @@
+# Logging
+
+In this document, we present the most common fields of our log messages.
+Remember that with structured logging all messages are represented as an array of key-value pairs,
+as in `{"key_1": "value_1", ..., "key_n": "value_n"}`.
+
+- `"module"`: the module in the ZetaClient that originated the message.
+              It can have one of the following values: “orchestrator”, “inbound”, “outbound”,
+              “signer”, “gas_price”, “btc_client”, “zetacore_client”, “tss_healhcheck”,
+              “tss_keygen”, “tss_keysign”, “tss_service”, and “tss_setup”.
+- `"chain"`: chain identifier (it can come paired with the `"chain_network"` field key).
+- `"block"`: height of a block.
+- `"nonce"`: nonce of a transaction from an external chain.
+- `"tx"`: hash of a transaction from an external chain.
+- `"txid"`: TXID from a Bitcoin transaction.
+- `"cctx"`: unique identifier (index) of a cross-chain transaction.
+- `"zeta_tx"`: hash of a ZetaChain transaction.
+- `"outbound_id"`: unique identifier of an outbound transaction (usually contains the nonce of the
+                   outbound in the external chain plus the chain identifier).
+- `"ballot"`: ballot from voting on a inbound or outbound cross-chain transaction.
+- `"coin_type"`: GAS, BTC, ERC20, SOL, ZETA, etc.
```

Comment:
Need to explain what it represents
In fact the coint_type is more what the type of asset in the CCTX is which is
- Gas
- ERC20 (fungible asset, might be renamed in the future)
- ZETA
- NoAssetCall, no asset transported in the cctx